src/hooks/spacewalk/useBridgeSettings.ts

```diff
@@ -1,8 +1,9 @@
 import _ from 'lodash';
-import { StateUpdater, useEffect, useMemo, useState } from 'react';
+import React, { StateUpdater, useEffect, useMemo, useState } from 'react';
```

Comment:
Should we import this from `preact`? I forgot if this makes a difference.

Reply:
I think we could be consistent with the imports (but they are not across the app in many places).  I don't think makes a big difference tbh.